Fix-C24_WMDE_Desktop_DE_01

Sperling-0 opened the pull request:
Ticket: https://phabricator.wikimedia.org/T366159

moiikana commented on pad/BannerVar.jsx:
```diff
@@ -0,0 +1,214 @@
+	// eslint-disable-next-line no-unused-vars
+	componentDidUpdate( previousProps, previousState, snapshot ) {
+		if ( previousState.formInteractionSwitcher !== this.state.formInteractionSwitcher ) {
+			this.adjustSurroundingSpace();
```
should this be `onPageResize` maybe ?

Sperling-0 committed:
Prepare campaign for C24_WMDE_Desktop_DE_02

Ticket: https://phabricator.wikimedia.org/T368509

Sperling-0 committed:
Prepare CTRL for C24_WMDE_Desktop_DE_02

- Update tests
- Fix bug in slide show content

Ticket: https://phabricator.wikimedia.org/T368509

Co-authored by: Abban Dunne <abban.dunne@wikimedia.de>

Sperling-0 committed:
Implement VAR for C24_WMDE_Mobile_DE_02

Sperling-0 committed:
Fix -- Make clock update in real time

Ticket: https://phabricator.wikimedia.org/T366159

Sperling-0 committed:
Fix -- Add tests for banners

Ticket: https://phabricator.wikimedia.org/T366159

Sperling-0 commented:
Base branch is wrong.
